clusters/build-clusters/01_cluster/Makefile

```diff
@@ -15,17 +16,17 @@ generate-install-config:
 	@echo "run 'cp /tmp/install-config.yaml ~/install-config.yaml' before installing the cluster"
 
 #TODO (hongkliu): rest api for lastest
-release_image := registry.svc.ci.openshift.org/ocp/release:4.2.0-0.nightly-2019-10-11-230724
+release_image := quay.io/openshift-release-dev/ocp-release:4.2.10
```

Comment:
```console
$ curl -sH 'Accept:application/json' 'https://api.stage.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.2&arch=amd64' | jq -r '.nodes[] | select(.version == "4.2.12")'
{
  "version": "4.2.12",
  "payload": "quay.io/openshift-release-dev/ocp-release@sha256:77ade34c373062c6a6c869e0e56ef93b2faaa373adadaac1430b29484a24d843",
  "metadata": {
    "description": "",
    "io.openshift.upgrades.graph.release.manifestref": "sha256:77ade34c373062c6a6c869e0e56ef93b2faaa373adadaac1430b29484a24d843",
    "url": "https://access.redhat.com/errata/RHBA-2019:4181",
    "io.openshift.upgrades.graph.release.channels": "candidate-4.2,fast-4.2,stable-4.2"
  }
}
```

4.2.12 is stable now.  Ideally this would just float with https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/ or (if you want to pin the minor version) https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.2/

Reply:
hacked api.stage's response to get the max value on tag.

Any way getting json from mirror urls above (instead of txt)?